docs/RESOURCES.md

```diff
@@ -0,0 +1,50 @@
+# PASS Tech Stack & Development Resources
+
+[![Javascript](https://badges.aleen42.com/src/javascript.svg)](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+[![Node.JS](https://badges.aleen42.com/src/node.svg)](https://nodejs.org/en)
+[![NPM](https://badges.aleen42.com/src/npm.svg)](https://www.npmjs.com/)
+[![React](https://badges.aleen42.com/src/react.svg)](https://react.dev/)
+[![Vite](https://badges.aleen42.com/src/vitejs.svg)](https://vitejs.dev/)
+[![ESLint](https://badges.aleen42.com/src/eslint.svg)](https://eslint.org/)
+[![Git-Hub](https://badges.aleen42.com/src/github.svg)](https://github.com/codeforpdx/PASS)
+[![Discord](https://badges.aleen42.com/src/discord.svg)](https://discord.com/)
+[![Google Meet](https://img.shields.io/badge/Google%20Meet-00897B?style=for-the-badge&logo=google-meet&logoColor=white)](https://meet.google.com/)
+[![Material UI](https://img.shields.io/badge/Material%20UI-007FFF?style=for-the-badge&logo=mui&logoColor=white)](https://mui.com/)
+<a href="https://solidproject.org"><img src="https://solidproject.org/assets/img/solid-emblem.svg" alt="SolidProjectIcon" width="18" height="18"> Solid Project
+<a href="https://www.inrupt.com/"><img src="https://docs.inrupt.com/inrupt-logo-small.svg" alt="InruptIcon" width="18" height="18"> Inrupt(solid library)
+<a href="https://eslint.org/"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e3/ESLint_logo.svg/324px-ESLint_logo.svg.png?20211012234406" alt="SolidProjectIcon" width="18" height="18"> ESLint
+🗒️  [JSDoc](https://jsdoc.app/)
+  ![tech stack](https://drive.google.com/uc?id=1hvs0XKI0hfdlGOJ_oRG-MqQTR9NCtmPy)
```

Comment:
Think we can split this into two sections, one for the shields, one for the image itself. Otherwise, it'll stretch out the entire section.

https://github.com/codeforpdx/PASS/assets/14917816/5dfff484-770b-48ed-aefd-83d34782fc21

Reply:
added it to a section